docs/migration.md

```diff
@@ -3,198 +3,107 @@ title: Migration
 sort_rank: 10
 ---
 
-# Prometheus 2.0 migration guide
+# Prometheus 3.0 migration guide
 
-In line with our [stability promise](https://prometheus.io/blog/2016/07/18/prometheus-1-0-released/#fine-print),
-the Prometheus 2.0 release contains a number of backwards incompatible changes.
-This document offers guidance on migrating from Prometheus 1.8 to Prometheus 2.0 and newer versions.
+In line with our [stability promise](https://prometheus.io/docs/prometheus/latest/stability/),
+the Prometheus 3.0 release contains a number of backwards incompatible changes.
+This document offers guidance on migrating from Prometheus 2.x to Prometheus 3.0 and newer versions.
 
 ## Flags
```

Comment:
Can we add `# Configuration` 

https://github.com/prometheus/prometheus/pull/15178 got renamed. While native histograms were experimental, it's worth mentioning the rename?